napari/conftest.py

```diff
@@ -470,3 +472,166 @@ def _mock_app():
             yield app
         finally:
             Application.destroy('test_app')
+
+
+def _get_calling_place(depth=1):
+    if hasattr(sys, "_getframe"):
+        frame = sys._getframe(1 + depth)
+        return f"{frame.f_code.co_filename}:{frame.f_code.co_firstlineno}"
+    return ""
+
+
+@pytest.fixture
+def dangling_threads(monkeypatch):
```

Comment:
Consider renaming to `dangling_qthreads` to clarify this only captures `QThreads` (e.g. and not threads started with Python's `threading`).

It might be worth add `q` or `qt` to all of the fixtures defined here to clarify that these are qt related.

Reply:
changed

Reply:
Looks good now.